Added additional functionality for the AttachedTo processing.

blackiceua opened the pull request:
Added additional functionality for the AttachedTo processing.
Improved AttachedTo processing.
Fixed typos.

blackiceua committed:
Added additional functionality for the AttachedTo processing.
Improved AttachedTo processing.
Fixed typos.

blackiceua committed:
Added handling for attaching note to element before minimization.

blackiceua committed:
Added processing for correct positioning of fixed minimized variant.

txusko reviewed (commented):
First, thanks for your submission 😄 
Could you please add a brief explanation, and what's the aim of this modification in the PR description? Maybe a screenshot or a GIF could explain better your changes.

blackiceua committed:
Disable default font import in CSS

The default import of the 'Open Sans' font from Google Fonts has been commented out in the Post-It-All CSS. This change allows for the disabling of the default browser tooltip font.
